Fix intro background color

DrGlaucous opened the pull request:
The intro scene should have a black background, not dark blue.

(fixed image)
![image](https://github.com/user-attachments/assets/ad54f692-ba2c-451d-9fcf-f1498313d877)

alula commented:
CS+ uses dark blue in some areas iirc and the Switch version doesn't use black at all, this is a bit of a shitty fix tbh

alula commented:
This shouldn't even be set in scene initialization but in engine constants for freeware and re-tested fully

DrGlaucous commented:
Where does CS+ not use black where Freeware does?
I haven't come across any differences when I checked. (The main two places are the intro and island scene, both use black)

You're right about the Switch port though. It uses blue everywhere.
I've addressed your concerns with my latest commit.
